assets/activation-scripts/etc/profile.d/0100_run-mode-paths.sh

```diff
@@ -58,27 +49,6 @@ export PATH
 MANPATH="$FLOX_ENV/share/man:${MANPATH:+$MANPATH}"
 export MANPATH
 
-# ---------------------------------------------------------------------------- #
-
-if [ -n "${FLOX_ENV_LIB_DIRS:-}" ]; then
-  case "$($_coreutils/bin/uname -s)" in
-    Linux*)
-      # N.B. ld-floxlib.so makes use of FLOX_ENV_LIB_DIRS directly.
-      LD_FLOXLIB="${LD_FLOXLIB:-@ld-floxlib@}"
-      if [ -z "${FLOX_NOSET_LD_AUDIT:-}" ] && [ -e "$LD_FLOXLIB" ]; then
-        LD_AUDIT="$LD_FLOXLIB"
-        export LD_AUDIT
-      fi
-      ;;
-    Darwin*)
-      if [ -z "${FLOX_NOSET_DYLD_FALLBACK_LIBRARY_PATH:-}" ]; then
-        DYLD_FALLBACK_LIBRARY_PATH="$FLOX_ENV_LIB_DIRS:${DYLD_FALLBACK_LIBRARY_PATH:-/usr/local/lib:/usr/lib}"
-        export DYLD_FALLBACK_LIBRARY_PATH
-      fi
-      ;;
-  esac
-fi
-
```

Comment:
@limeytexan noted in [this thread](https://flox-dev.slack.com/archives/C05P6A5J6U8/p1731046427662569) that this is needed for run mode because:

> Otherwise the binaries created in the build wouldn't be able to find their shared libraries.